sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobQuerySerialization.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+/**
+ * Defines the input and output serialization for a blob quick query request.
+ * either {@link BlobQueryJsonSerialization} or {@link BlobQueryDelimitedSerialization}
+ */
+public abstract class BlobQuerySerialization {
+
+    protected char recordSeparator;
```

Comment:
@seanmcc-msft discovered that .NET XMLSerializer transform line endings. We should check if Java's XML engine does the same thing and if it does then we'd probably need to ask to extend generator.
For example `\n` might be converted into `\r\n`. We should make sure whitespaces are sent to server side verbatim.

Other thing to check is column separator in delimited serialization. For example adding test that would query a [TSV](https://en.wikipedia.org/wiki/Tab-separated_values) file.

Reply:
Yep! Just added a few tests for this. It seems like \t works. Some that don't work however, include \r, \n. I'll bring this up in standup